Dockerfile

```diff
@@ -1,6 +1,5 @@
-FROM ghcr.io/navikt/baseimages/temurin:17-appdynamics
+FROM ghcr.io/navikt/baseimages/temurin:21
```

Comment:
har de ikke appdynamics lenger?

Reply:
usikker, appd skal jo fases ut snart uansett. så tenkte jeg droppet det. https://nav-it.slack.com/archives/C01DE3M9YBV/p1695905476412559

Reply:
La inn appdynamics igjen siden opentelemetry er ett lenger lærret å bleke...